contents/handbook/product/metrics.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Product metrics
+sidebar: Handbook
+showTitle: true
+---
+
+As we improve our Product, it's key to have a guiding metric that we can all at Product & Engineering rally behind and measure our progress against. For the Product, this metric is "SNAZZY NAME HERE". The metric can be tracked in the [product dasboard][dashboard] and is reported weekly at PostHog News.
+
+> 💡 "SNAZZY NAME HERE" is different from Discovered Learnings. Discovered Learnings is used as a metric for activation and growth.
+
+Generally, small teams should be making measurable impact towards improving this metric (not all teams, not all the time). It's okay for epics and sprints to have other intermediate (and probably more concrete) goals, but overall each team should be moving the metric as a whole.
+
+## Metric definition
+The metric is defined [in this action][action]. The concept of the metric is users driving insights from PostHog, so any of these actions:
+- Analyzing any insight. Analyzing means viewing for 10 seconds or more. Insights include: trends, funnels, sessions (*), paths, lifecycle, stickiness and quant analysis.
```

Comment:
How do you propose we add new insights to this definition when they're created?

Reply:
Added a note